lab-james/src/app/reducer.js

```diff
@@ -0,0 +1,13 @@
+const initialState = [];
+
+export default (state=initialState, action) => {
+  let {type, payload} = action;
+
+  switch(type){
+
+  case 'NOTE_ADD': return [...state, payload];
```

Comment:
hmm, you can indent switches this way but it's usually better to have it another level in.